README.md

```diff
@@ -6,8 +8,8 @@
 | Dependency updater | [![Dependency Updater Status](https://github.com/nix-community/nix-doom-emacs/workflows/Update%20Dependencies/badge.svg?branch=master)](https://github.com/nix-community/nix-doom-emacs/actions/workflows/update-dependencies.yml?query=branch%3Amaster) |
 | Matrix Chat | [![Matrix Chat](https://img.shields.io/static/v1?label=chat&message=doom-emacs&color=brightgreen&logo=matrix)](https://matrix.to/#/#doom-emacs:nixos.org) |
 
-Nix expression to install and configure
-[doom-emacs](https://github.com/doomemacs/doomemacs).
+nix-doom-emacs (abbreviated as NDE) provides a customisable Nix derivation for [Doom Emacs](https://github.com/doomemacs/doomemacs)
+The project has lots of moving pieces and hacks. Users are expected to know their way around using (and especially debugging) Nix and Emacs Lisp in order to use this project.
```

Comment:
There wasn't a double newline and I think the second sentence is kind of gatekeepy. I think people can judge it for themselves–sorry for the changed opinion.
```suggestion
nix-doom-emacs (abbreviated as NDE) provides a customizable Nix derivation for [Doom Emacs](https://github.com/doomemacs/doomemacs).
```